lib/ex_libsrtp.ex

```diff
@@ -0,0 +1,108 @@
+defmodule ExLibSRTP do
+  @moduledoc """
+  [ExLibSRTP](https://github.com/cisco/libsrtp) bindings for Elixir.
```

Comment:
We probably don't want to rename this one, want we?
```suggestion
  [libsrtp](https://github.com/cisco/libsrtp) bindings for Elixir.
```